fastparquet/util.py

```diff
@@ -106,3 +122,21 @@ def index_like(index):
                 index._start == 0 and
                 index._stop == len(index) and
                 index._step == 1 and index.name is None)
+
+
+def check_column_names(columns, *args):
+    """Ensure that parameters listing column names have corresponding columns"""
+    for arg in args:
+        if isinstance(arg, (tuple, list)):
```

Comment:
If that's not part of the PR then my comments are probably OT. Though I must add that what this function is supposed to do, and why it is coded as it is, is still a mystery to me.